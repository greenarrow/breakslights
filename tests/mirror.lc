MR001
MA001
A000 F00ff00 B000000 IF S002 P001 D255 Z1
R000 N000
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
