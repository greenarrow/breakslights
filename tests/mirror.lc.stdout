00 00ff00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 00ff00
00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 000000 000000 00ff00 00ff00
00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00
00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00
00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00
00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00
00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
00 00ff00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 00ff00
00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 000000 000000 00ff00 00ff00
00 00ff00 00ff00 00ff00 000000 000000 000000 000000 000000 000000 00ff00 00ff00 00ff00
00 00ff00 00ff00 00ff00 00ff00 000000 000000 000000 000000 00ff00 00ff00 00ff00 00ff00
00 00ff00 00ff00 00ff00 00ff00 00ff00 000000 000000 00ff00 00ff00 00ff00 00ff00 00ff00
00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00 00ff00
00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
